cognite/client/_api/simulators/models_revisions.py

```diff
@@ -0,0 +1,209 @@
+from __future__ import annotations
+
+from collections.abc import Iterator, Sequence
+from typing import TYPE_CHECKING, Any, NoReturn, overload
+
+from cognite.client._api_client import APIClient
+from cognite.client._constants import DEFAULT_LIMIT_READ
+from cognite.client.data_classes._base import CogniteFilter
+from cognite.client.data_classes.simulators.filters import SimulatorModelRevisionsFilter
+from cognite.client.data_classes.simulators.models import (
+    CreatedTimeSort,
+    SimulatorModelRevision,
+    SimulatorModelRevisionList,
+    SimulatorModelRevisionWrite,
+)
+from cognite.client.utils._experimental import FeaturePreviewWarning
+from cognite.client.utils._identifier import IdentifierSequence
+from cognite.client.utils._validation import assert_type
+from cognite.client.utils.useful_types import SequenceNotStr
+
+if TYPE_CHECKING:
+    from cognite.client import ClientConfig, CogniteClient
+
+
+class SimulatorModelRevisionsAPI(APIClient):
+    _RESOURCE_PATH = "/simulators/models/revisions"
+
+    def __init__(self, config: ClientConfig, api_version: str | None, cognite_client: CogniteClient) -> None:
+        super().__init__(config, api_version, cognite_client)
+        self._warning = FeaturePreviewWarning(
+            api_maturity="General Availability", sdk_maturity="alpha", feature_name="Simulators"
+        )
+        self._CREATE_LIMIT = 1
+        self._RETRIEVE_LIMIT = 1
+
+    def list(
+        self,
+        limit: int = DEFAULT_LIMIT_READ,
+        sort: CreatedTimeSort | None = None,
+        filter: SimulatorModelRevisionsFilter | dict[str, Any] | None = None,
+    ) -> SimulatorModelRevisionList:
+        """`Filter simulator model revisions <https://developer.cognite.com/api#tag/Simulator-Models/operation/filter_simulator_model_revisions_simulators_models_revisions_list_post>`_
+        Retrieves a list of simulator model revisions that match the given criteria
+        Args:
+            limit (int): Maximum number of results to return. Defaults to 25. Set to -1, float(“inf”) or None to return all items.
+            sort (CreatedTimeSort | None): The criteria to sort by.
+            filter (SimulatorModelRevisionsFilter | dict[str, Any] | None): Filter to apply.
+        Returns:
+            SimulatorModelRevisionList: List of simulator model revisions
+        Examples:
+            List simulator model revisions:
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> res = client.simulators.models.revisions.list()
+
+            Specify filter and sort order:
+                >>> from cognite.client.data_classes.simulators.filters import SimulatorModelRevisionsFilter
+                >>> from cognite.client.data_classes.simulators.models import CreatedTimeSort
```

Comment:
I think it would be nice if these are importable from `data_classes.simulators` directly:
```suggestion
                >>> from cognite.client.data_classes.simulators import SimulatorModelRevisionsFilter
                >>> from cognite.client.data_classes.simulators import CreatedTimeSort
```

Reply:
done :)